app/service/callback_rest.py

```diff
@@ -47,6 +48,19 @@ def fetch_service_callback(
     return jsonify(data=service_callback_api_schema.dump(service_callback)), 200
 
 
+@service_callback_blueprint.errorhandler(409)
+def handle_conflict(error):
+    return jsonify(message=str(error.description)), 409
```

Comment:
This might be okay, but the other error handlers are [here](https://github.com/department-of-veterans-affairs/notification-api/blob/main/app/__init__.py#L397).  Should a 409 behave differently for other routes?  If not, this code probably doesn't belong here.